src/main/java/picard/cmdline/CommandLineProgram.java

```diff
@@ -282,13 +319,48 @@ protected boolean parseArgs(final String[] argv) {
     }
 
     public String getStandardUsagePreamble() {
-        return standardUsagePreamble;
+        return getCommandLineParser().getStandardUsagePreamble(getClass());
     }
 
+    /**
+     * @return Return the command line parser to be used.
+     */
     public CommandLineParser getCommandLineParser() {
+        if (commandLineParser == null) {
+            commandLineParser = useLegacyParser(getClass()) ?
+                        new LegacyCommandLineArgumentParser(this) :
+                        new CommandLineArgumentParser(this,
+                            Collections.EMPTY_LIST,
+                            new HashSet<>(Arrays.asList(CommandLineParserOptions.APPEND_TO_COLLECTIONS)));
```

Comment:
Better yet replace the expression with `Collections.singleton(CommandLineParserOptions.APPEND_TO_COLLECTIONS))` or
`EnumSet.of(CommandLineParserOptions.APPEND_TO_COLLECTIONS)`. Both assume that `CommandLineArgumentParser` doesn't side effect its arguments, which it shouldn't.